micro-service/src/utils/index.js

```diff
@@ -0,0 +1,11 @@
+export const getChurchSlug = (host) => {
+  let subdomain =
+    process.env.NODE_ENV === 'production'
+      ? host.split('.').slice(0, -2).join('.')
+      : process.env.NEXT_APP_CHURCH_SLUG;
```

Comment:
now you can provide a custom church slug via the `NEXT_PUBLIC_CHURCH_SLUG` header.